lib/client-assets.php

```diff
@@ -790,7 +790,7 @@ function gutenberg_editor_scripts_and_styles( $hook ) {
 	$script  = '( function() {';
 	$script .= sprintf( 'var editorSettings = %s;', wp_json_encode( $editor_settings ) );
 	$script .= <<<JS
-		window._wpLoadGutenbergEditor = Promise.all( [ wp.api.init(), wp.api.init( { versionString: 'gutenberg/v1' } ) ] ).then( function() {
+		window._wpLoadGutenbergEditor = Promise.all( [ wp.api.init(), wp.api.init( { versionString: 'gutenberg/v1/' } ) ] ).then( function() {
```

Comment:
curious, what's the reason for this change?

Reply:
None of the Backbone methods work without the trailing slash. We didn't catch this while testing https://github.com/WordPress/gutenberg/pull/3377 since the unit tests mock the API call and no UI existed for any of this.

Reply:
Do you think this is still useful with https://github.com/WordPress/gutenberg/pull/3778

Reply:
Yes—we're not using `withAPIData`.